lib/radiator_web/live/outline_component.ex

```diff
@@ -228,11 +228,9 @@ defmodule RadiatorWeb.OutlineComponent do
   defp indent(socket, uuid, prev_id) do
     user_id = socket.assigns.user_id
 
-    Dispatch.move_node(uuid, user_id, generate_event_id(socket.id),
-      parent_id: prev_id,
-      prev_id: nil
-    )
+    Dispatch.indent_node(uuid, user_id, generate_event_id(socket.id))
 
+    # TODO prev_id might not be nil!!
```

Comment:
this is tricky. prev id might be the last element in the list (not nil) if new parent already contains children